src/screening/sputum/SputumForm.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { formValueSelector, change, untouch } from 'redux-form';
+import { Obs, formUtil, selectors } from '@openmrs/react-components';
+import { Grid, Row, FormGroup, ControlLabel, Col } from 'react-bootstrap';
+import { ENCOUNTER_TYPES, CONCEPTS, FORM_ANSWERS } from "../../constants";
+import ScreeningForm from "../ScreeningForm";
+import { setFlex, padding } from "../../pwaStyles";
+
+class SputumForm extends React.PureComponent {
+
+  componentDidUpdate(prevProps) {
+    // this clears out form values when the "Sputum received" question is changed
+    if (typeof this.props.sputumReceived !== 'undefined' && this.props.sputumReceived !== prevProps.sputumReceived) {
+      if (this.props.sputumReceived === CONCEPTS.SputumReceivedFalse.uuid) {
+        this.props.dispatch(change(this.props.formInstanceId, formUtil.obsFieldName('sputum-sample-quality', CONCEPTS.SampleQuality.uuid), null));
+        this.props.dispatch(untouch(this.props.formInstanceId, formUtil.obsFieldName('sputum-sample-quality', CONCEPTS.SampleQuality.uuid)));
+
+        this.props.dispatch(change(this.props.formInstanceId, formUtil.obsFieldName('sputum-laboratory-location', CONCEPTS.LabLocation.uuid), null));
+        this.props.dispatch(untouch(this.props.formInstanceId, formUtil.obsFieldName('sputum-laboratory-location', CONCEPTS.LabLocation.uuid)));
+      }
+    }
+  }
+
+  render() {
+    const formContent = (
+      <Grid>
+        <Row>
+          <Col
+            componentClass={ControlLabel}
+          >
+            Sputum received
+          </Col>
+        </Row>
+        <Row>
+          <Col sm={12}>
+            <FormGroup controlId="formBled">
+              <Obs
+                concept={CONCEPTS.SputumReceived.uuid}
+                conceptAnswers={FORM_ANSWERS.trueFalse}
```

Comment:
@mogoodrich , HERE I had to use the Yes/No concept Answers and one you posted on the ticket. That one wasn't working. Is this fine?

Reply:
@jomadoye yes, that is correct, to use the true/false ones... I will comment to Dave on the ticket